tests/functional/characterisation/framework.sh

```diff
@@ -63,7 +63,7 @@ function characterisationTestExit() {
             echo ''
             echo 'You can rerun this test with:'
             echo ''
-            echo "    _NIX_TEST_ACCEPT=1 make tests/functional/${TEST_NAME}.sh.test"
+            echo "    _NIX_TEST_ACCEPT=1 meson test lang"
```

Comment:
This seemed to work in https://github.com/NixOS/nix/pull/12372/files#diff-36724f93532488ab13c7db3ea46b7849517158c13888e2811965d18c8bcd4a6bR74

```suggestion
            echo "    _NIX_TEST_ACCEPT=1 meson test ${TEST_NAME}"
```